client/src/modules/stock/movements/registry.html

```diff
@@ -50,53 +81,34 @@
           </ul>
         </div>
       </div>
-
-      <div class="toolbar-item">
-        <bh-renderer-dropdown
-          report-url="/reports/stock/movements"
-          report-options="StockCtrl.filters">
-        </bh-renderer-dropdown>
-      </div>
     </div>
   </div>
 </div>
 
-<!-- util bar  -->
-<div class="flex-util">
-  <bh-filters-applied
-    style="max-width:90%"
-    filters="StockCtrl.formatedFilters"
+<div class="flex-util bh-filter-bar">
+  <bh-filters
+    filters="StockCtrl.latestViewFilters"
     on-remove-filter="StockCtrl.onRemoveFilter(filter)">
-  </bh-filters-applied>
-
-  <a
-    href
-    ng-if="!StockCtrl.isToday"
-    ng-click="StockCtrl.clearFilters()"
-    class="text-danger"
-    data-method="clear">
-    <i class="fa fa-ban text-danger"></i>
-    <span translate>FORM.INFO.CLEAR_FILTERS</span>
-  </a>
+  </bh-filters>
 </div>
 
 <!--the grid -->
 <div class="flex-content">
   <div class="container-fluid">
-
     <div
       id="stock-movements-grid"
       ui-grid="StockCtrl.gridOptions"
       class="grid-util-full-height"
       ui-grid-resize-columns
       ui-grid-auto-resize
+      ui-grid-auto-resize
```

Comment:
It looks like you duplicated `ui-grid-auto-resize` here.  This should be removed since I think it sets up two `$interval` loops that will negatively impact performance.